configurations/generic/credhub.yml

```diff
@@ -2,9 +2,9 @@
   path: /releases/-
```

Comment:
Could you please compare this section with `bosh-deployment/credhub.yml` and make it closer to that.
I would really-really like to use community version later.